packages/SwingSet/test/snapshots/test-xsnap-store.js.md

```diff
@@ -20,18 +20,18 @@ Generated by [AVA](https://avajs.dev).
 
     {
       compressSeconds: 0,
-      hash: '29278a07d019b09317376597a2bc9d942551f988863a8f34789757205c38b662',
+      hash: 'f9c184a555e80f08e85768187f525909e630079a82d6bd29b0e391e1e53d5c25',
```

Comment:
This changed because filenames such as `types-ambient.js` have runtime impact, I suppose.

Reply:
yeah or the removal of `// @ts-check`